src/PrestaShopBundle/Controller/Admin/Improve/Design/CmsPageController.php

```diff
@@ -970,211 +1049,136 @@ private function getCmsPageFormHandler()
-            return $this->trans(
-                'Can\'t delete #%id%',
-                'Admin.Notifications.Error',
-                [
-                    '%id%' => $exception->getCmsPageCategoryId(),
-                ]
-            );
-        }
-
-        $exceptionType = get_class($exception);
-        if (isset($exceptionTypeDictionary[$exceptionType])) {
-            return $exceptionTypeDictionary[$exceptionType];
-        }
-
-        return $this->getFallbackErrorMessage($exceptionType, $exception->getCode());
-    }
-
-    /**
-     * Gets exception of cms page category by its type and status code.
-     *
-     * @param CmsPageCategoryException $exception
-     *
-     * @return string
-     */
-    private function getCmsPageCategoryErrorByExceptionTypeAndCode(CmsPageCategoryException $exception)
-    {
-        $exceptionTypeDictionary = [
+            CannotDisableCmsPageException::class => $this->trans(
+                'An error occurred while updating the status.',
+                'Admin.Notifications.Error'
+            ),
+            CannotEnableCmsPageException::class => $this->trans(
+                'An error occurred while updating the status.',
+                'Admin.Notifications.Error'
+            ),
+            CannotDeleteCmsPageException::class => [
+                CannotDeleteCmsPageException::FAILED_BULK_DELETE => $this->trans(
+                    'An error occurred while deleting this selection.',
+                    'Admin.Notifications.Error'
+                ),
+                CannotDeleteCmsPageException::FAILED_DELETE => $this->trans(
+                    'An error occurred while deleting the object.',
+                    'Admin.Notifications.Error'
+                ),
+            ],
             CmsPageCategoryConstraintException::class => [
                 CmsPageCategoryConstraintException::INVALID_BULK_DATA => $this->trans(
                     'You must select at least one element to delete.',
                     'Admin.Notifications.Error'
                 ),
                 CmsPageCategoryConstraintException::CANNOT_MOVE_CATEGORY_TO_PARENT => $this->trans('The page Category cannot be moved here.', 'Admin.Design.Notification'),
                 CmsPageCategoryConstraintException::MISSING_DEFAULT_LANGUAGE_FOR_NAME => $this->trans(
-                        'The %s field is not valid',
-                        'Admin.Notifications.Error',
-                        [
-                            sprintf(
-                                '"%s"',
-                                $this->trans('Name', 'Admin.Global')),
-                        ]
-                    ),
+                    'The %s field is not valid',
+                    'Admin.Notifications.Error',
+                    [
+                        sprintf(
+                            '"%s"',
+                            $this->trans('Name', 'Admin.Global')),
```

Comment:
Weird place for `)` at the end. Same for other items in this array. I think you could inline it.